src/Npgsql/PostgresErrorCodes.cs

```diff
@@ -0,0 +1,338 @@
+﻿namespace Npgsql
+{
+    /// <summary>
+    /// Provides constants to PostgreSQL's error codes.
```

Comment:
nit: "Provides constants for PostgreSQL error codes."

Reply:
done.